config/pmix_check_cflags.m4

```diff
@@ -40,3 +40,14 @@ AC_MSG_CHECKING(if $CC supports ([$1]))
                 AC_MSG_RESULT([yes])
             fi
 ])
+
+AC_DEFUN([_PMIX_CHECK_LTO_FLAG], [
+    chkflg=`echo $1 | grep -- -flto`
+    if test "$chkflg"; then
```

Comment:
```suggestion
    if test -n "$chkflg"; then
```

I can't remember offhand if `test "$foo"` is the same as `test -n "$foo"`.

I'm 100% sure that `-n` means "check for a non-empty string".  I don't remember offhand if having no test operator means "check for a true-ish value" or "check for a non-empty string".

Reply:
It is the same, but no harm in adding the `-n` to make it clear.